pkg/db/query/payload_queries.go

```diff
@@ -57,13 +57,13 @@ func GetLastOSUpgradeByArchitectureAndStream(db *gorm.DB, release string) ([]mod
 }
 
 // GetLastPayloadTags returns payloads tags for last two weeks, sorted by date in descending order.
-func GetLastPayloadTags(db *gorm.DB, release, stream, arch string) ([]models.ReleaseTag, error) {
+func GetLastPayloadTags(db *gorm.DB, release, stream, arch string, timeNow time.Time) ([]models.ReleaseTag, error) {
```

Comment:
Ah to my earlier comment, here we're passing gorm.DB not our db.DB, but we could likely adjust it to take a db.DB pretty easily.